docs/geedocs/answer/7160000.html

```diff
@@ -210,6 +296,41 @@
           <td>Portable Globe Cutter fails with a Python error</td>
           <td>Fixed import errors in the cutter scripts</td>
         </tr>
+        <tr>
+          <td>381</td>
+          <td>Error running POI Search</td>
+          <td>Needed to declare the coding in headers of python script:# -- coding: utf-8 -</td>
+        </tr>
+        <tr>
+          <td>382</td>
```

Comment:
I think these three documentation fixes (382, 413, and 414) can be removed.  We have a statement at the top (in the paragraph about open sourcing) that says a number of documentation changes were made.  I think that covers this, though I'd be open to other's thoughts.

Reply:
Yeah if these doc cleanup 'bugs' are not worth mentioning, i can remove them.

Reply:
I vote to remove them.

Reply:
Done.

Reply:
I could also see leaving just 382 as a sort of catch all, but you might want to update the issue number to "382 and others" or something like that.  If you try to list them all it will be a long list I think.